packages/authentication/android/src/main/java/dev/robingenz/capacitorjs/plugins/firebase/authentication/handlers/FacebookAuthProviderHandler.java

```diff
@@ -58,20 +59,23 @@ public void onError(FacebookException exception) {
         }
     }
 
-    public void signIn(PluginCall call) {
-        savedCall = call;
-        JSArray scopes = call.getArray("scopes");
+    private void applySignInConfig() {
```

Comment:
```suggestion
    private void applySignInConfig(LoginButton loginButton) {
```

Please pass the button instance instead of accessing it directly.